external/vcm/vcm/catalog.yaml

```diff
@@ -665,8 +665,9 @@ sources:
 
   2020_1yr_pire_pressure_interpolated:
     description: >-
-      3D dynamical core diagnostics variables from 40-day nudged simulation (May 27 2020 SHiELD), coarsened to C48 resolution and interpolated to pressure levels.
-      Generated with https://github.com/VulcanClimateModeling/explore/blob/master/annak/2021-06-17-save-c3072-verification-correct-winds.ipynb
```

Comment:
It might be good to leave the trail of notebooks used to create this for provenance.